NEWS

```diff
@@ -172,8 +172,12 @@ Changed Functionality
 - The Notice framework's suppression implementation was changed to batch suppression
   information for up up to 10msec before distributing it to other nodes in a Zeek cluster.
   This reduces cluster communication overhead when there's a storm of notices
-  generated. Two new options, ``Notice::suppression_batch_period`` and ``Notice::suppression_batch_max_size``
-  can be used to tune this behavior. See their documentation for details.
+  generated. Two new options, ``Notice::suppression_batch_period`` and
+  ``Notice::suppression_batch_max_size`` can be used to tune this behavior. See their
+  documentation for details.
+
+- Non-printable ASCII control characters are no longer escaped with an ``\\x`` prefix in
```

Comment:
Maybe turn the wording around to strengthen that all JSON logging is affected by this one, possibly with an example?